### PR TITLE
87965 Import document definition

### PR DESCRIPTION
**Pull request by theo-ritense**

### CODE REVIEW

File: case/src/main/kotlin/com/ritense/case/web/rest/CaseDefinitionResource.kt [hunk 1]
```diff
@@ -182,8 +183,8 @@ class CaseDefinitionResource(
     @RunWithoutAuthorization
     fun import(
         @RequestParam("file") file: MultipartFile
-    ): ResponseEntity<Unit> {
-        return try {
+    ): ResponseEntity<Unit> = runWithoutAuthorization {
```
Comment (Klaas-Ritense) [inline case/src/main/kotlin/com/ritense/case/web/rest/CaseDefinitionResource.kt]: 2x runWithoutAuthorization

File: document/src/main/kotlin/com/ritense/document/service/JsonSchemaDocumentDefinitionImporter.kt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import com.ritense.document.service.impl.JsonSchemaDocumentDefinitionService
+import com.ritense.importer.ImportRequest
+import com.ritense.importer.Importer
+import org.springframework.transaction.annotation.Transactional
+
+@Transactional
+class JsonSchemaDocumentDefinitionImporter(
+    private val jsonSchemaDocumentDefinitionService: JsonSchemaDocumentDefinitionService
+) : Importer {
+    override fun type() = "form-definition"
```
Comment (Klaas-Ritense) [inline document/src/main/kotlin/com/ritense/document/service/JsonSchemaDocumentDefinitionImporter.kt]: form-definition?

File: document/src/test/kotlin/com/ritense/document/service/JsonSchemaDocumentDefinitionImporterTest.kt [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        assertThat(importer.dependsOn()).isEmpty()
+    }
+
+    @Test
+    fun `should support document definition fileName`() {
+        assertThat(importer.supports(FILENAME)).isTrue()
+    }
+
+    @Test
+    fun `should not support non-formflow fileName`() {
```
Comment (Klaas-Ritense) [inline document/src/test/kotlin/com/ritense/document/service/JsonSchemaDocumentDefinitionImporterTest.kt]: formflow?

File: case/src/main/kotlin/com/ritense/case/web/rest/CaseDefinitionResource.kt (hunk 1 above)
Comment (theo-ritense) [reply case/src/main/kotlin/com/ritense/case/web/rest/CaseDefinitionResource.kt]: Ah, I overlooked the `@RunWithoutAuthorization`. Fixed

File: document/src/main/kotlin/com/ritense/document/service/JsonSchemaDocumentDefinitionImporter.kt (hunk 2 above)
Comment (theo-ritense) [reply document/src/main/kotlin/com/ritense/document/service/JsonSchemaDocumentDefinitionImporter.kt]: fixed

File: document/src/test/kotlin/com/ritense/document/service/JsonSchemaDocumentDefinitionImporterTest.kt (hunk 3 above)
Comment (theo-ritense) [reply document/src/test/kotlin/com/ritense/document/service/JsonSchemaDocumentDefinitionImporterTest.kt]: fixed

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**